core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportProfileData.java

```diff
@@ -108,6 +113,42 @@ public void testImportMutationsFile() throws Exception {
         validateMutationAminoAcid (geneticProfileId, secondSampleId, 2842, "L113P");
     }
 
+    @Test
+    public void testImportGermlineOnlyFile() throws Exception {
+        /* Mutations file split over two files with same stable id */
+        String[] args = {
+                "--data","src/test/resources/germlineOnlyMutationsData/data_mutations_extended.txt",
+                "--meta","src/test/resources/germlineOnlyMutationsData/meta_mutations_extended.txt",
+                "--loadMode", "bulkLoad"
+        };
+        ImportProfileData runner = new ImportProfileData(args);
+        runner.run();
+        String studyStableId = "study_tcga_pub";
+        CancerStudy study = DaoCancerStudy.getCancerStudyByStableId(studyStableId);
+        studyId = study.getInternalId();
+        int sampleId = DaoSample.getSampleByCancerStudyAndSampleId(studyId, "TCGA-AA-3664-01").getInternalId();
+        GeneticProfile geneticProfile = DaoGeneticProfile.getGeneticProfileByStableId(studyStableId + "_breast_mutations");
+        geneticProfileId = geneticProfile.getGeneticProfileId();
+
+        // assume clinical data for MUTATION_COUNT was created
+        ClinicalAttribute clinicalAttribute = DaoClinicalAttributeMeta.getDatum("MUTATION_COUNT", studyId);
+        assertNotNull(clinicalAttribute);
```

Comment:
this is the key test ... that the MUTATION_COUNT attribute was computed for the import of this all-GERMLINE (for this sample) profile